lib/src/header_parser/utils.dart

```diff
@@ -197,8 +197,9 @@ String? removeRawCommentMarkups(String? string) {
   string = string.replaceAll('/*', '');
   string = string.replaceAll('*/', '');
 
-  // Remove any *'s in the beginning of a every line.
+  // Remove any *'s or // in the beginning of a every line.
```

Comment:
This didn't cause any of our existing large tests to get different output? Did you regenerate all of those?

If not, maybe we should add some more small manual tests with different comment styles.

Reply:
The large tests use ** style so it didn't cause any different output. We are testing the comments under large tests. LibClang tests - doxygen, full and SQLIte tests any, full comments.

Reply:
I'll add new tests and update the way we remove comment markup using suggestions from Jonas.

Reply:
Done.